includes/Plugins.php

```diff
@@ -205,6 +218,53 @@ final class Plugins {
 		),
 	);
 
+	/**
+	 * Initial plugins to be installed based on the capabilities set by Hiive
+	 *
+	 * @var array
+	 */
+	public static $hiive_flag_plugin_list = array(
+		'hasEcomdash'     => array(
+			array(
+				'slug'     => 'nfd_slug_ecomdash_wordpress_plugin',
+				'activate' => true,
+				'priority' => 20,
```

Comment:
I think priority makes sense in relation to other plugins in the queue. We need to assign priority only once the final list of plugins based on the capability is populated. Otherwise, it doesn't make sense to assign random priority numbers. Please check how this is being done in the plugin recommendation install flow and see if we can replicate the same approach.